package.json

```diff
@@ -6,7 +6,8 @@
   "license": "Apache-2.0",
   "repository": "adobe/aio-cli-plugin-asset-compute",
   "dependencies": {
-    "@adobe/aio-lib-runtime": "^2.0.0",
+    "@adobe/aio-cli-plugin-app": "^8.0.0",
```

Comment:
@adobe/aio-cli-lib-app-config has been split out into its own module so you no longer need a dependency on the entire app-plugin ( its big with webpack, and parcel dependencies )